src/api/serviceCalls.ts

```diff
@@ -27,3 +27,18 @@ export const getEmployeeDetails = async (onSuccess, onError) => {
       onError(err.message);
     });
 };
+
+export const getAllEmployeeMetrics = async (onSuccess, onError) => {
+  await axios
+    .get('/api/employee', {
```

Comment:
the endpoint for getting all the employee metrics is employee/hours/all, is this being rerouted somewhere else where it hits that endpoint?